include/fmt/chrono.h

```diff
@@ -449,6 +459,11 @@ template <typename Char> struct formatter<std::tm, Char> {
       const size_t MIN_GROWTH = 10;
       buf.reserve(buf.capacity() + (size > MIN_GROWTH ? size : MIN_GROWTH));
     }
+
+    if (writeSubseconds) {
+      buf.append(std::to_string(subseconds));
```

Comment:
This does not add leading zeroes.

Reply:
Thank you for the comments. I did notice I've overlooked some of those, but I was out. Should be fixed now.